src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -298,21 +298,21 @@ const UsersFragment = ({
 const styles = StyleSheet.create({
   fragmentContainer: {
     flex: 1,
-    backgroundColor: COLORS.white
+    backgroundColor: COLORS.almostBlack
   },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color of the `fragmentContainer` style has been changed from `COLORS.white` to `COLORS.almostBlack`. Ensure that this change is intentional and aligns with the overall design scheme. If it's a mistake, revert it back.
```diff
-    backgroundColor: COLORS.almostBlack
+    backgroundColor: COLORS.white
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->